masterlist.yaml

```diff
@@ -228,11 +233,16 @@ common:
   - &reqBetterMales
     <<: *reqModX
     subs: [ '[Better males - Beautiful nudes and faces - New hairstyles](http://www.nexusmods.com/skyrim/mods/2488)' ]
-
-  - &reqSkyUI # Useful for when a plugin needs SkyUI, but shouldn't load after it (eg. when it's a master plugin).
+  # Useful for when a plugin needs SkyUI, but shouldn't load after it (eg. when it's a master plugin).
```

Comment:
Any non-cosmetic reason for this move? (I don't disagree with the move! Just curious. :))